src/explorer.api/Controllers/ExploreController.cs

```diff
@@ -103,14 +93,28 @@ public class ExploreController : ControllerBase
                 }
             }
 
+            var columnMetricsList = exploration.ColumnExplorations.Select(ce =>
+                new ExploreResult.ColumnMetricsCollection(
+                    ce.Column,
+                    ce.PublishedMetrics.Select(m =>
+                        new ExploreResult.Metric(m.Name, m.Metric))));
```

Comment:
I think the logic for building the `ExploreResult` from the completed Exploration should be in the ExploreResult class itself rather than in the Controller. 

Maybe a static initialiser `FromSuccessfulExploration` or something like that.

Reply:
Fixed with db0b3fdb6f932011ef529ed170b3b84cb3a10d34. 
Would it make sense to also rename `ExploreResult` to `ExplorationResult`?

Reply:
It's the result of a call to the `/explore` endpoint so  `ExploreResult` fits fine IMO. But I have no strong feeling about this either way.